src/python/pants/backend/codegen/thrift/java/java_thrift_library.py

```diff
@@ -77,6 +80,10 @@ def rpc_style(self):
     return self._rpc_style
```

Comment:
I think that it might be a good idea to deprecate `rpc_style` in this same change, as discussed.

To do that, you can use `deprecated_conditional` ([an example usage](https://github.com/pantsbuild/pants/blob/ae1ea88d5a280120dff3a4cff2141b774ccca139/src/python/pants/backend/python/targets/python_target.py#L40)) in the target `__init__` constructor to trigger a warning if someone passes `rpc_style`.